README.md

```diff
@@ -43,6 +43,8 @@ Santiment API python client.
     - [Top Holders Percent of Total Supply](#top-holders-percent-of-total-supply)
     - [History Twitter Data](#history-twitter-data)
     - [Price Volume Difference](#price-volume-difference)
+    - [Ethereum Top Transactions](#ethereum-top-transactions)
```

Comment:
why do we have this one in `news` PR ?

Reply:
I added the metric to the already updated master (the Ethereum Top Transactions was the last PR)

Reply:
This one should not be here, maybe you need to rebase against master?

Reply:
Ethereum Top Transactions is the last PR request, doesn't that mean that it is already in master (the last PR is merged) ?